jest.config.js

```diff
@@ -0,0 +1,23 @@
+const nextJest = require('next/jest')
+
+const createJestConfig = nextJest({
+  // Provide the path to your Next.js app to load next.config.js and .env files in your test environment
+  dir: './',
+})
+
+// Add any custom config to be passed to Jest
+const customJestConfig = {
+  setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
+  moduleNameMapper: {
+    // Handle module aliases (this will be automatically configured for you soon)
+    '^@/components/(.*)$': '<rootDir>/components/$1',
+
+    '^@/pages/(.*)$': '<rootDir>/pages/$1',
+  },
+  testEnvironment: 'jest-environment-jsdom',
+}
+
+// createJestConfig is exported this way to ensure that next/jest can load the Next.js config which is async
+module.exports = createJestConfig(customJestConfig)
+// import '@testing-library/jest-dom';
```

Comment:
What's going on here?

Reply:
That is an excellent question. That entire `config` file was a copy/paste job. I don't actually know what most of it is doing. If I just comment out lines 19-21, my tests still pass so...I might not need it.

Reply:
Ooo...sorry. I meant specifically yon line 22. That lil comment.

Reply:
I don't see why I would need to import anything there. I can remove all of that stuff towards the bottom, lines 19-22 and tests still pass. Doesn't look like I need it, or at the very least I don't see how getting rid of that code would affect us long term.

Reply:
Yeah that's what I'm saying. I think that you added this accidentally and commented it out. Line 22 should be removed.